tensorflow_io/core/kernels/azfs_kernels.cc

```diff
@@ -95,7 +120,8 @@ Status ParseAzBlobPath(StringPiece fname, bool empty_object_ok,
     *object = std::string(objectp.substr(pos + 1));
```

Comment:
I think the check for `empty_object_ok == True` condition is missing.

Reply:
Thanks @kvignesh1420. Updated to capture `empty_object_ok == True`